sympy/core/tests/test_assumptions.py

```diff
@@ -767,7 +768,8 @@ def test_special_is_rational():
     assert (sqrt(3) + S.Pi).is_rational is False
     assert (x**i).is_rational is None
     assert (i**i).is_rational is True
-    assert (r**i).is_rational is True
+    assert (i**i2).is_rational is None
+    assert (r**i).is_rational is None
```

Comment:
The generic tests that are here cover the code added, don't they? If so, this is fine.